uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab-filters.component.js

```diff
@@ -99,5 +94,13 @@ define([
         return typeA.title.localeCompare(typeB.title) ? typeA : typeB;
       });
     }
+
+    /**
+     * Emits the *filtersUpdated* event, passing the filter values to the parent
```

Comment:
A bit of inconsistency:
- *filtersUpdated*
- "filters updated" (in another file)
I wonder which one is correct?